hooks/useCopyToClipboard.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+
+const copyToClipboard = (value: string) => {
+  if (typeof window === 'undefined') {
+    return Promise.resolve(false);
+  }
+
+  return navigator.clipboard
+    .writeText(value)
+    .then(() => true)
+    .catch(() => false);
+};
+
+// eslint-disable-next-line no-unused-vars
+export const useCopyToClipboard = (): [boolean, (text: string) => void] => {
```

Comment:
Could you remove this explicit return type and then remote the eslint-disable statement

```suggestion
export const useCopyToClipboard = () => {
```